### PR TITLE
Reworked some CABAL weapons and diverse other changes

### PR DESCRIPTION
**Pull request by trempler**
Added new explosion system for GDI,NOD,CABAL structures
Added new tesla weapon to sabercut and changed it to anti infantry
Added new tesla weapon for dreadfire
Added new laserweapon to core obelisk (Pew pew)
Added new laserlogic to obelisk of darkness (Prism aoe)
Added blue explo for all cabal units.
Added new explosions for ref,harv,silo
Added new abillity totalmeltdown for cabal
Added detection description for venom,hydra,zodiac
Added gasclouds for devils tongue and hailstorm

Fixed visionrange of skirmisher
Fixed GDI service depot hitbox 1x1>3x3

Changed skirmisher ammo
Changed cabal speed upgrade.
Changed cloakingfield of golem down to 3c512
Changed dreadfire name to shockwave
Changed repair Intervall from 6 down to 8
Changed plasma blaster to anti vehicle and adjusted the fx
Changed costs from golem 800$ to 900$
Changed reaper weapon to fit cabal more (clusterrocket)
Changed avatar to no more aoe dmg
Changed zodiac movementspeed from 35 down to 25
Changed fanatic to never explode on death.
Changed cooldown of shadowtrooper drop 3:00 to 4:00
Changed cooldown of napalmstrike 5:00 to 4:00 also reduced direct dmg to vehicle
Changed cooldown of tiberiumbomb 5:00 to 4:00 and reduced direct dmg to buildings
Changed subterranean apc ambush changed e3 count up to 5 and reduced nasol to 3
Changed subapc of industry nod to radartech
Changed raiderbuggy available for all nod factions
Changed artillerystrike to deal 20%less dmg, adjusted aoe, reduced cd from 5:00 to 4:00
Changed emergencyrepairs to repair 30% slower
Changed orca bombingrun to deal 20%less dmg and reduced cd from 5:00 to 4:00
Changed ox-sonic bombing with increased movement and adjusted cd, 3:30 to 2:30

Removed attack bike cloak detection
Removed tiberiumspawn with harvexplo

### CODE REVIEW

**Comment by trempler** [comment]
sry, for other branch still included, forgot to delete it.

**Comment by GraionDilach** [comment]
I also don't see fixes for anything I pointed out yesterday.

**Comment by jrb0001** [comment]
This has to be rebased onto the update branch after https://github.com/DoGyAUT/cd/pull/145 has been merged.

File: rules/cabal-vehicles.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 KMCV:
 	Inherits@1: ^VoxelVehicle
 	Inherits@2: ^HoverVehicle
+	Inherits@3: ^VehicleC
```
Comment (jrb0001) [inline rules/cabal-vehicles.yaml]: What does `VehicleC` mean? Should probably be renamed.

File: rules/defaults.yaml [hunk 2]
```diff
@@ -1587,6 +1616,14 @@
 	ExternalCondition@INVULNERABILITY:
 		Condition: invulnerability
 
+^VehicleC:
+	Explodes:
+		Weapon: UnitExplodeSmallC
```
Comment (jrb0001) [inline rules/defaults.yaml]: What does `UnitExplodeSmallC` mean? Should probably be renamed.

File: rules/defaults.yaml [hunk 3]
```diff
@@ -1587,6 +1616,14 @@
...
+		EmptyWeapon: UnitExplodeSmallC
+	ThrowsShrapnel@debris:
+		Weapons: SmallDebrisc
```
Comment (jrb0001) [inline rules/defaults.yaml]: What does `SmallDebrisc` mean? Should probably be renamed.

File: weapons/shared-weapons.yaml [hunk 4]
```diff
@@ -88,7 +88,13 @@ UnitExplodeSmall:
 	Warhead@4Eff: CreateEffect
 		Explosions: medium_flash
 		ExplosionPalette: light2
-
+		
+UnitExplodeSmallC:
```
Comment (jrb0001) [inline weapons/shared-weapons.yaml]: What does `UnitExplodeSmallC` mean? Should probably be renamed.

File: weapons/shared-weapons.yaml [hunk 5]
```diff
@@ -88,7 +88,13 @@ UnitExplodeSmall:
...
+	Inherits@1: UnitExplodeSmall
+	Warhead@2Eff: CreateEffect
+		Explosions: blueexpl01, blueexpl02, blueexpl03, blueexpl04
+		ExplosionPalette: effect25alpha
+
```
Comment (jrb0001) [inline weapons/shared-weapons.yaml]: Whitespace on empty line.

File: weapons/shared-weapons.yaml [hunk 6]
```diff
@@ -285,6 +291,54 @@ LargeVeinExplosion:
 		Explosions: tibbomb
 		ExplosionPalette: effect50alpha
 		ImpactSounds: expnew09.aud
+
```
Comment (jrb0001) [inline weapons/shared-weapons.yaml]: Whitespace on empty line.

**Review by jrb0001 — CHANGES_REQUESTED** [review]
In addition to the other comments, naming (https://github.com/DoGyAUT/cd/issues/148) has to be fixed too. Because that has to be done in the refactoring anyway, I suggest moving it to later.

File: rules/cabal-vehicles.yaml (hunk 1 above)
Comment (jrb0001) [reply rules/cabal-vehicles.yaml]: Done.

File: rules/defaults.yaml (hunk 2 above)
Comment (jrb0001) [reply rules/defaults.yaml]: Done.

File: rules/defaults.yaml (hunk 3 above)
Comment (jrb0001) [reply rules/defaults.yaml]: Done.

File: weapons/shared-weapons.yaml (hunk 4 above)
Comment (jrb0001) [reply weapons/shared-weapons.yaml]: Done.

File: weapons/shared-weapons.yaml (hunk 5 above)
Comment (jrb0001) [reply weapons/shared-weapons.yaml]: Done.

File: weapons/shared-weapons.yaml (hunk 6 above)
Comment (jrb0001) [reply weapons/shared-weapons.yaml]: Done.

**Review by jrb0001 — APPROVED** [review]

**Review by CDVoidwalker — APPROVED** [review]
Looks good to me

**Pull request merged**